unified_planning/engines/sequential_simulator.py

```diff
@@ -283,6 +289,11 @@ def supported_kind() -> "up.model.ProblemKind":
         supported_kind.set_effects_kind("INCREASE_EFFECTS")
         supported_kind.set_effects_kind("DECREASE_EFFECTS")
         supported_kind.set_simulated_entities("SIMULATED_EFFECTS")
+        supported_kind.set_quality_metrics("ACTIONS_COST")
+        supported_kind.set_quality_metrics("PLAN_LENGTH")
+        supported_kind.set_quality_metrics("OVERSUBSCRIPTION")
+        supported_kind.set_quality_metrics("MAKESPAN")
+        supported_kind.set_quality_metrics("FINAL_VALUE")
```

Comment:
This change is related to the comment in the #297

Reply:
Yes, I think this also has to be discussed in a meeting: probably the simulator will ignore quality metrics, but the validator I have no idea, still has to be discussed at all

Reply:
If we add the quality metrics here, we should add them also in the sequential plan validator